nvd3/templates/piechart.html

```diff
@@ -69,6 +69,15 @@
         chart.color(mycolor);
     {% endif %}
 
+    {% block rendering_opts %}
+        {% if chart.no_data_message %}
```

Comment:
This is written out instead of just calling `super` because pie chart cannot have the `stacked` option but it can have the `no_data_message` and `show_controls` options. If we're okay with including `stacked` (and just count on people not trying to apply it to pie chart` I can revise this to just call `super()`

Reply:
good with me!